feather/server/src/packet_handlers.rs

```diff
@@ -128,3 +129,16 @@ fn handle_chat_message(game: &Game, player: EntityRef, packet: client::ChatMessa
     game.broadcast_chat(ChatKind::PlayerChat, message);
     Ok(())
 }
+
+fn handle_client_settings(
+    server: &mut Server,
+    player: EntityRef,
+    packet: client::ClientSettings,
+) -> SysResult {
+    /*log::trace!("Updating a player's client settings!");*/
```

Comment:
Either remove this or replace it with one we can merge. It would have to include which player if you wanted to keep it in.

Reply:
you mean the commented out trace right?

Reply:
Yes! I'm not sure why it doesn't make that more clear.